skfda/exploratory/visualization/_ddplot.py

```diff
@@ -0,0 +1,139 @@
+"""DD-Plot Module.
+
+This module contains the necessary functions to construct the DD-Plot.
+To do this depth is calculated for the two chosen distributions, and then
+a scatter plot is created of this two variables.
+"""
+
+from typing import List, Optional, TypeVar
+
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+from ...exploratory.depth.multivariate import Depth
+from ._utils import _get_figure_and_axes, _set_figure_layout_for_fdata
+
+T = TypeVar('T')
+S = TypeVar('S', Figure, Axes, List[Axes])
```

Comment:
This should not be a `TypeVar` as it is only used once.

Reply:
should i do then instead the S with a Union?

Reply:
Yes.

Reply:
can you please review the code? I think now everything is fine